code/modules/mining/machine_processing.dm

```diff
@@ -11,7 +11,7 @@
 	var/machinedir = EAST
 	var/show_all_ores = 0
 
-/obj/machinery/mineral/processing_unit_console/New()
+/obj/machinery/mineral/processing_unit_console/New()/*
```

Comment:
Just comment the entire proc, not just the contents

Reply:
Understood. I shall do that in the morning tomorrow.

Reply:
Or better yet, purge it